src/host/getset.cpp

```diff
@@ -269,7 +269,8 @@ void ApiRoutines::GetNumberOfConsoleMouseButtonsImpl(ULONG& buttons) noexcept
         consoleFontInfoEx.FontFamily = fontInfo.GetFamily();
         consoleFontInfoEx.FontWeight = fontInfo.GetWeight();
 
-        RETURN_IF_FAILED(StringCchCopyW(consoleFontInfoEx.FaceName, ARRAYSIZE(consoleFontInfoEx.FaceName), fontInfo.GetFaceName()));
+        // GH#3123: We will fail to service the API if the font name is longer than 32 characters.

```

Comment:
Ehhh can we just truncate it to safely fit? I feel that's better than failing.

Reply:
Maybe we want a method that's like "GetNameGdiCompatible()" that fills an LF_FACESIZE size array.